### DISCUSSION TITLE
Office warning

### DISCUSSION THREAD

**Discussion by steamdeckuserr** [OdysseusYuan/Mocreak#29] [Q&A] [answered]
Hi, recently, when I open office, the warning shows like the attached picture. How to solve it? Thanks!
![warning](https://github.com/user-attachments/assets/246a74c7-7d9f-41be-bc97-a0f0b4f14bd8)

**Comment by OdysseusYuan** [comment]
Thank you for your use. Can you tell me your Office version? 2016 or 2019?

**Reply by steamdeckuserr** [reply]
Thank you for the reply. I installed the 2024 newest ver.

**Reply by OdysseusYuan** [reply, marked as answer]
You can try downloading the latest version of the tool, which may solve your problem.
https://github.com/OdysseusYuan/Mocreak/releases/latest
